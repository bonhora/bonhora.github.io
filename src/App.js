// import logo from './logo.svg';

import { BrowserRouter as Router, Routes, Route } from "react-router-dom";

import "./normalize-2.css";

import Articulos from "./components/Articulos/Articulos";
import Header from "./components/Header/Header";
import Footer from "./components/Footer/Footer";
import ErrorPage from "./components/ErrorPage/ErrorPage";
import ArticulosPage from "./components/ArticuloPage/ArticulosPage";
import Login from "./components/Login/Login";
import ScrollToTop from "./components/ScrollToTop/ScrollToTop";
import Register from "./components/Register/Register";
import AboutUs from "./components/AboutUs/AboutUs";
import ContactoPage from "./components/ContactoPage/ContactoPage";




function App() {
  return (
    <Router>
      <div className="App">
        <Header />

        <ScrollToTop>
          <Routes>
            <Route path="/" element={<Articulos />} />
            <Route path="/articulos/:id" element={<ArticulosPage />} />
            <Route path="/login" element={<Login />} />
            <Route path="/register" element={<Register />} />
            <Route path="/about-us" element={<AboutUs />} />
            <Route path="/contact" element={<ContactoPage />} />

              
            <Route path="*" element={<ErrorPage />} />
          </Routes>
        </ScrollToTop>

        <Footer />
      </div>
    </Router>
  );
}

export default App;
