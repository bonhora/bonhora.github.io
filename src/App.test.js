import { render, screen } from '@testing-library/react';
import App from './App';
import Enzyme, { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

it("renders correctly", () => {
  const wrapper = shallow(<App />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
