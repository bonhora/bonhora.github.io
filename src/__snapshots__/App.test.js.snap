// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
  </header>
</div>
`;
